FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -0,0 +1,100 @@
+//
+//  CoreDataFeedStore.swift
+//  FeedStoreChallenge
+//
+//  Created by Shilpa Bansal on 23/02/21.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import CoreData
+
+public class CoreDataFeedStore: FeedStore {
+	private let persistentContainer: NSPersistentContainer
+	private let managedContext: NSManagedObjectContext
+	public static let modelName = "FeedStoreDataModel"
+	
+	public init(storeURL: URL?) throws {
+		let storeBundle = Bundle(for: CoreDataFeedStore.self)
+		
+		guard let model = NSManagedObjectModel.with(name: CoreDataFeedStore.modelName, in: storeBundle) else {
+			throw NSError(domain: "Bundle URL is nil11111", code: 0, userInfo: nil)
```

Comment:
The error message can be improved to clarify the error

```suggestion
			throw NSError(domain: "Couldn't find object model in Bundle", code: 0, userInfo: nil)
```